packages/editor/src/components/block-list/insertion-point.js

```diff
@@ -88,18 +87,15 @@ export default withSelect( ( select, { clientId, rootClientId } ) => {
 	const {
 		getBlockIndex,
 		getBlockInsertionPoint,
-		getBlock,
 		isBlockInsertionPointVisible,
 	} = select( 'core/editor' );
 	const blockIndex = getBlockIndex( clientId, rootClientId );
 	const insertIndex = blockIndex;
 	const insertionPoint = getBlockInsertionPoint();
-	const block = getBlock( clientId );
 	const showInsertionPoint = (
 		isBlockInsertionPointVisible() &&
 		insertionPoint.index === insertIndex &&
-		insertionPoint.rootClientId === rootClientId &&
-		! isUnmodifiedDefaultBlock( block )
```

Comment:
This basically says, we don't show the insertion point indicator if the block is empty (empty paragraph block). I don't really understand why personally in addition to the fact that it harms the performance. @jasmussen Thoughts on that? How does the insertion indicator works on this PR for you?